docs-client/src/containers/App/index.tsx

```diff
@@ -372,7 +372,7 @@ class App extends React.PureComponent<Props, State> {
               color="inherit"
               noWrap
             >
-              Armeria documentation service
+              Armeria documentation service {specification.getVersion()}
```

Comment:
I found out that we do not display the snapshot part when building site. Could you remove it as well, please? See https://github.com/line/armeria/blob/master/site/src/sphinx/conf.py#L37

Reply:
I believe it's because we do not want to expose `-SNAPSHOT` in the official web site. I think a user should be given with full information in this case.